src/GoTrueClient.ts

```diff
@@ -920,6 +935,16 @@ export default class GoTrueClient {
         Boolean(getParameterByName('error_description')))
     )
   }
+  /**
+   * Checks if the current URL and backing storage contain parameters given by a PKCE flow
+   */
+  private async _isPKCEFlow(): Promise<boolean> {
+    const currentStorageContent = await getItemAsync(
+      this.storage,
+      `${this.storageKey}-code-verifier`
```

Comment:
```suggestion
      `${this.storageKey}-oauth-code-verifier`
```

Reply:
Ah, good catch! Have renamed everything else to `code-verifier` instead so as to keep it generic